bitwarden_license/src/Commercial.Core/SecretsManager/Commands/ServiceAccounts/AutoscaleServiceAccountsCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Bit.Core.Models.Business;
+using Bit.Core.OrganizationFeatures.OrganizationSubscriptionUpdate.Interface;
+using Bit.Core.SecretsManager.Commands.ServiceAccounts.Interfaces;
+
+namespace Bit.Commercial.Core.SecretsManager.Commands.ServiceAccounts;
+
+public class AutoscaleServiceAccountsCommand : IAutoscaleServiceAccountsCommand
+{
+    private readonly IUpdateSecretsManagerSubscriptionCommand _updateSecretsManagerSubscriptionCommand;
+
+    public AutoscaleServiceAccountsCommand(
+        IUpdateSecretsManagerSubscriptionCommand updateSecretsManagerSubscriptionCommand)
+    {
+        _updateSecretsManagerSubscriptionCommand = updateSecretsManagerSubscriptionCommand;
+    }
+
+    public async Task AutoscaleServiceAccountsAsync(Guid organizationId, int serviceAccountsToAdd)
+    {
+        await _updateSecretsManagerSubscriptionCommand.UpdateSecretsManagerSubscription(
+            new SecretsManagerSubscriptionUpdate
+            {
+                OrganizationId = organizationId,
+                SmServiceAccountsAdjustment = serviceAccountsToAdd
+            });
```

Comment:
You're right that this should just be a method on the update subscription command. It fits there (it's updating the subscription) and also commands shouldn't call other commands.

Reply:
I've moved it to `UpdateSecretsManagerSubscriptionCommand`